src/puppetlabs/puppetdb/command/dlo.clj

```diff
@@ -1,210 +1,182 @@
 (ns puppetlabs.puppetdb.command.dlo
-  (:import [org.apache.commons.io FileUtils])
```

Comment:
The DLO changes substantially here - discarded messages mirror the stockpile files they come from, and for each foo.json discarded command, there is a foo-err.txt detailing the full backtrace for each attempt.

Support for archiving, and per-command subdirs is gone.